specification/dartLangSpec.tex

```diff
@@ -5247,6 +5247,9 @@ \section{Mixins}
 A mixin describes the difference between a class and its superclass.
 A mixin is either derived from an existing class declaration
 or introduced by a mixin declaration.
+It is a compile-time error to derive a mixin from
+a class that explicitly declares a generative constructor,
```

Comment:
Is "explicitly" needed? Can you *declare* a generative constructor implicitly, or do we reserve the word "declare" for when you actually write something.

Reply:
I thought about the same thing: We don't otherwise say 'explicitly' (along with that, we don't consider any implicitly induced declarations to be part of the program), so the consistent choice is to remove the word `explicitly` here. Done.